### PR TITLE
Starting template for study set 4

### PR DESCRIPTION
**Pull request by shemaiah-s**
issue #9 
Hi Dr. Spielman, this is what I have so far. I will change the format at the end so that the questions for each set come first, followed by the answer key for that set. Do you have any changes you would like me to make going forward?

### CODE REVIEW

**Comment by shemaiah-s** [comment]
Hi Dr. Spielman, I just pushed over my latest commit and do not see it showing up in this chat.

**Comment by sjspielman** [comment]
Are you sure you pushed to the right branch? EDIT: I see you definitely did.. let's give the website a couple more minutes to work out, maybe it's stuck somewhere in internet ether.. I'll sleuth in the meantime.

**Comment by sjspielman** [comment]
Can you send a screen shot of the output from `git status` and `git log` (you can press `q` to exit out of the git log view)?

**Comment by sjspielman** [comment]
> issue #9
> Hi Dr. Spielman, this is what I have so far. I will change the format at the end so that the questions for each set come first, followed by the answer key for that set. Do you have any changes you would like me to make going forward?

Sorry I had missed this - *my* notifications weren't turned on!! Silly me...

**Comment by shemaiah-s** [comment]
Screen Shot 2020-07-15 at 12 34 56 PM
So this is git status

**Comment by shemaiah-s** [comment]
I cannot really screenshot all of the git log. Was there a specific section you would like to see?

**Comment by sjspielman** [comment]
> I cannot really screenshot all of the git log. Was there a specific section you would like to see?

Just the top part with any most recent commits that *should* be here. `git log` shows all commits ever made.

**Comment by shemaiah-s** [comment]
Screen Shot 2020-07-15 at 12 40 25 PM
Is this what you are looking for?

**Comment by sjspielman** [comment]
See above - indeed, github was being slow and silly!! The commits look like they're here now. Do you see anything missing?

**Comment by shemaiah-s** [comment]
Yes! I just reloaded and they are there now :) Also, I can reformat the questions and answer sections now. I just didn't know if you wanted to look at it before I reformatted it.

File: learnr_bank/learnr_set4.Rmd [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+#### 3. Use `select()` to subset `msleep` to *remove* the columns `genus` and `order`
+
+```{r dplyr1_q3, exercise = TRUE}
+
+```
+
+``` {r, dplyr1_q3-answer}
+#Solution to Question 3
+msleep %>%
+  select(- c(genus, order))
```
Comment (sjspielman) [inline learnr_bank/learnr_set4.Rmd]: `select(-genus, -order)`

File: learnr_bank/learnr_set4.Rmd [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+
+```{r dplyr1-2_q3, exercise = TRUE}
+
+``` 
+
+``` {r, dplyr1-2_q3-answer}
+#Solution to Question 3
+msleep %>%
+  filter(order == "Primates", conservation == "lc") %>%
+  select(-c(conservation, order)) %>%
```
Comment (sjspielman) [inline learnr_bank/learnr_set4.Rmd]: `select(-conservation, -order)`
Your solution is great and also works, but I just want to be consistent with how I'm introducing in the slides. Maybe you could include both solutions to show options?

File: learnr_bank/learnr_set4.Rmd [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+
+```{r dplyr1-2_q4, exercise = TRUE}
+
+``` 
+
+``` {r, dplyr1-2_q4-answer}
+#Solution to Question 4
+msleep %>%
+  filter(order == "Primates", conservation == "lc") %>%
+  select(-c(conservation, order)) %>%
```
Comment (sjspielman) [inline learnr_bank/learnr_set4.Rmd:297]: same thing here

File: learnr_bank/learnr_set4.Rmd [hunk 4]
```diff
@@ -0,0 +1,448 @@
...
+#### 1. Use `dplyr::summarize()` and `mean()` to determine the average amount of time spent awake (column `awake`) by all mammals in the dataset `msleep`.
+
+```{r dplyr2_q1, exercise = TRUE}
+
+``` 
+
+``` {r, dplyr2_q1-answer}
+#Solution to Question 1
+msleep %>%
+  summarise(mean(awake))
```
Comment (sjspielman) [inline learnr_bank/learnr_set4.Rmd]: we want to give this column a name, as in `summarize(mean_awake = mean(awake))`. Also, use the "z" and not "s" in summarize for consistency w/ directions.

File: learnr_bank/learnr_set4.Rmd [hunk 5]
```diff
@@ -0,0 +1,448 @@
...
+``` 
+
+<div id= "dplyr2_q2-hint">
+**Hint:** There are NA's in this column, so you need to tell the function `mean()` to ignore NA's with the extra argument `na.rm = TRUE`. Remember that?!
+</div> 
+
+``` {r, dplyr2_q2-answer}
+#Solution to Question 2
+msleep %>%
+  summarise(mean(sleep_rem, na.rm = TRUE))
```
Comment (sjspielman) [inline learnr_bank/learnr_set4.Rmd]: same here - give summarize columns names

**Comment by sjspielman** [comment]
Reformat away!! Looks great!! I added some comments in for specific lines of code too. None is wrong at ALL though, just a matter of style and consistency w/ the slides I'm presenting.
Also, can you make sure to copy the question itself into the answer key section? Thanks!!

**Comment by shemaiah-s** [comment]
Will the comments show up here with the code? It may still be a little slow because I haven't seen them yet.

**Review by sjspielman — CHANGES_REQUESTED** [review]
> Will the comments show up here with the code? It may still be a little slow because I haven't seen them yet.

This an excellent point... They will show up here and in the "Files Changed" tab. I see it all on my end, but maybe it hasn't made it to you? Don't worry about the comments yet - start with reorganizing and hopefully they'll appear by the time you're done. Comments are VERY small and easy.

**Comment by shemaiah-s** [comment]
I got them all. I also just pushed another commit. Hopefully, it shows up on your end soon.

**Review by sjspielman — APPROVED** [review]

**Comment by sjspielman** [comment]
Got it all, merging in!!

**Pull request merged**